doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
 <a name="2.2"></a>
 ## 2.2 Reference Model Requirements
 
-Traceability to Reference Model.
+This reference model intends to implement both the Basic, Network Intensive and Compute Intensive Instance types defined in the Reference Model. The following table contains the requirements of the Reference Model and indicates if the Reference Architecture supports them.
+
+<a name="2.2.1"></a>
+### 2.2.1 Network Acceleration Extensions
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| .il-ipsec | In-line IPSec acceleration | optional <sup>1)</sup> |
+| .la-crypto | Look-Aside encryption/decryption engine | optional <sup>2)</sup> |
+| .la-trans | Look-Aside Transcoding acceleration | optional <sup>3)</sup> |
+| .la-programmable | Look-Aside programmable acceleration | optional <sup>4)</sup> |
+
+**1)** In Kubernetes based infrastructures virtio-ipsec can be applied only if the CNI plugin is OVS.<br>
+**2)** In Kubernetes based infrastructures virtio-crypto can be applied only if the CNI plugin is OVS.<br>
+**3)** In Kubernetes based infrastructures virtio-trans can be applied only if the CNI plugin is OVS.<br>
+**4)** In Kubernetes based infrastructures virtio-programmable can be applied only if the CNI plugin is OVS.
+
+
+<a name="2.2.2"></a>
+### 2.2.2 Network Interface Options
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| n1, n2, n3, n4, n5, n6 | | Y |
+| n10, n20, n30, n40, n50, n60 | | Y |
+| n25, n50, n75, n100, n125, n150 | | Y |
+| n50, n100, n150, n200, n250, n300 | | Y |
+| n100, n200, n300, n400, n500, n600 | | Y |
+
+<a name="2.2.3"></a>
+### 2.2.3 VIM Capabilities
```

Comment:
@CsatariGergely Should the Traceability matrix be restricted to what is in scope for RA-2 only and not for example, VIM?

Reply:
@pgoyal01 In bare metal Kubernetes case Kubernetes is the VIM.